oak_functions_containers_app/src/lib.rs

```diff
@@ -380,6 +380,10 @@ pub async fn serve<G: AsyncRecipientContextGenerator + Send + Sync + 'static>(
     encryption_key_handle: Arc<G>,
     meter: Meter,
 ) -> anyhow::Result<()> {
+    // Tonic limits the incoming RPC size to 4 MB by default; bump it up to 1 GiB. We're not sending
+    // traffic over a "real" network anyway, after all.
+    let max_decoding_message_size = 1 * 1024 * 1024 * 1024;
```

Comment:
This could probably be a const somewhere, and declared in a nicer way with units, but does not have to block this PR.